test/unit/object/conftest.py

```diff
@@ -271,11 +271,28 @@ def if_none_match_header(etag):
     return {'If-None-Match': etag} if etag is not None else None
 
 
+@pytest.fixture()
+def if_match_sha1_header(etag, sha1):
```

Comment:
lol, of course we factored this code out in the test code, not production code.

Reply:
Included it here because it is consistent with how the tests abstract header creation.